dhall/man/dhall.md

```diff
@@ -0,0 +1,107 @@
+% dhall (1)
+
+# NAME
+
+dhall - a programmable configuration language
+
+# DESCRIPTION
+
+**dhall** is the command-line interface to the Dhall language
+
+# SYNOPSIS
+
+  dhall -\-file check.dhall
+
+  dhall -\-file error.dhall -\-explain
+
+  dhall hash -\-file to-hash.dhall
```

Comment:
Another useful command that a user might commonly use is `dhall type --quiet` to just check that import resolution and type-checking succeed